hooks/dependencyInstaller.js

```diff
@@ -0,0 +1,15 @@
+console.log("Running hook to install hooks pre-requisites");
```

Comment:
I know this 🔨  has been used many times before but we must start to refrain ourselves from using it as this contributes to bloating the source code :(